include/zephyr/drivers/bluetooth.h

```diff
@@ -0,0 +1,257 @@
+/** @file
+ *  @brief Bluetooth HCI driver API.
+ *
+ *  Copyright (c) 2024 Johan Hedberg
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ */
```

Comment:
It is not clear to me whether this API is experimental, stable or unstable. 
Suggest to add a version to it as per https://docs.zephyrproject.org/latest/develop/api/overview.html#api-overview to indicate it's mature-ness

Reply:
> It is not clear to me whether this API is experimental, stable or unstable. Suggest to add a version to it as per https://docs.zephyrproject.org/latest/develop/api/overview.html#api-overview to indicate it's mature-ness

@carlescufi can you help out here? I think probably "unstable" is the right categorization. It can't be experimental at least, since we couldn't deprecate the old API otherwise.

Reply:
Definitely "unstable", correct, no other definition fits this specific usecase. This API is new but already has multiple implementations and is the successor to an established one.